onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -942,7 +942,7 @@ def _dequantize_value(self, value_name):
                 self.model.model.producer_name == "onnx-quantizer" and scale_init is not None
             ):
                 # axis is not specified so scale_init must be a scalar.
-                assert onnx.numpy_helper.to_array(scale_init).size == 1
+                assert scale_init is None or onnx.numpy_helper.to_array(scale_init).size == 1
```

Comment:
Can you explain in what situation `scale_init` would be `None`?

Reply:
I did not write the tool. I only refactored it. Given the test just above the assert, scale_init can be None so the assert is wrong. But I don't a simple model in mind where this happens.

Reply:
Could you add a unit test for it? I tried to use onnx.utils.extract_model and onnxconverter_common.onnx2py but cannot come up a small unit test.

Reply:
I'll see what i can do. I did not add one because the assert is clearly wrong in of the cases the test allows. I could also fix the test and leave the assert as it is.

Reply:
I did not have time to add a unit test. I suggest merging it anyway. The assert is clearly not consistent with the test just before it right now.

Reply:
Would merge it.